src/fastapi_cli/cli.py

```diff
@@ -45,6 +48,13 @@ def callback(
     """
 
 
+def _get_docs_url(uvicorn_path: str) -> str:
+    module_path, app_name = uvicorn_path.split(sep=":")
+    module = importlib.import_module(module_path)
+    app = getattr(module, app_name)
+    return app.docs_url or DEFAULT_DOCS_URL
```

Comment:
Doesn't `docs_url is None` mean the docs are disabled.

Reply:
I couldn't figure out the difference between the `docs_url` field not being informed or this same field being set to `None`. I ended up changing it only for cases where the value was informed, otherwise I kept the same value that was hardcoded.

Reply:
@killjoy1221 I modified the code to omit the `Api doc` section if `docs_url=None` is explicitly specified. Please review.